opc/resource_storage_container.go

```diff
@@ -0,0 +1,182 @@
+package opc
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/go-oracle-terraform/client"
+	"github.com/hashicorp/go-oracle-terraform/storage"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceOPCStorageContainer() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceOPCStorageContainerCreate,
+		Read:   resourceOPCStorageContainerRead,
+		Delete: resourceOPCStorageContainerDelete,
+		Update: resourceOPCStorageContainerUpdate,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"read_acls": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
```

Comment:
Does this have any ordering issues, as the parent type is a list? ie: does the API return the same items, but in a different order?